test/binCases/version/mixed/precedence-command.test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
In these tests you are mixing commands and the test doesn't help to understand what's the expected outcome. Using the snapshot is not enough. I'd suggest to add to those test more assertion about what's correct what is not

Reply:
I was planning to use snapshots for version and help flags, because they are static, will not change much. In hindsight, it might be beneficiary to assert correct versions, yes. 

The convention I've used to diff these two are the naming, but they need a more consistent and clear separation, indeed. 

Do you have any suggestion for the naming?